### ISSUE TITLE
[$10][MSFT-128] Sample challenge page: The 'submit' link appears grayed out visually but it is not announced as disabled by the screen reader.

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#2826]
**Steps to Reproduce**
1. Go to the sample challenges page: https://clicktime.symantec.com/34CaebazimgmFxdkBZnihRc7Vc?u=https%3A%2F%2Fwww.topcoder.com%2Fchallenges%2F<sample-Sample challenge page:challengeid>?tab=details. 2. Ensure that the screen reader is switched on. 3. Navigate to the 'submit' link using the arrow keys. 4. Notice that this link appears greyed out visually however it is not announced as disabled by the screen reader.

**Expected Result**
The screen reader should announce the 'submit' link as disabled when it is not available for activation. Colour should not be the only way to highlight the element as disabled.

**Actual Result**
The screen reader doesn't announce the 'submit' link as disabled even though it is visually greyed out.

**Device**: Desktop/Labtop
**Operating System**: Windows
**Browser**: Chrome

**WCAG Category**: 1. Perceivable
**WCAG Level**: A
**WCAG Success Criterion**: 1.4.1 Use of Color
**Screen Reader Used**: JAWS
**Accessibility Audit Tool used**: N/A

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=088f5f5d%2D9b1c%2D4b7e%2Dacbf%2D320c257c9a9a&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2FKameshwarid%2Fbug%2032

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30095995 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30095995 has been updated - it has been assigned to PkDurlabhji.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by PrakashDurlabhji** [comment]
@veshu kindly read description in PR. thankyou 
https://github.com/topcoder-platform/topcoder-react-utils/pull/35

**Comment by cwdcwd** [comment]
[400]: Failed to create challenge. Detail: Operation failed in the contest service facade.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[400]: Failed to create challenge. Detail: Operation failed in the contest service facade.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[400]: Failed to create challenge. Detail: Operation failed in the contest service facade.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[400]: Failed to create challenge. Detail: Operation failed in the contest service facade.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097259 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097259 has been updated - it has been assigned to PkDurlabhji.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by PrakashDurlabhji** [comment]
@veshu new PR https://github.com/topcoder-platform/topcoder-react-utils/pull/48

**Comment by crazyk07** [comment]
Closing ticket due to shift in priority, will not be able to complete but will still pay for effort rendered.

Note: Ticket was dependent on react Utils

**Comment by crazyk07** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30097259<br/><br/>```This is an automated message for crazyk via Topcoder X```
